js/src/components/paid-ads/budget-setup/budget-badge.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Internal dependencies
+ */
+import Badge from '~/components/badge';
+import DeltaValue from '~/components/delta-value';
+import round from '~/utils/round';
+import styles from './budget-setup.module.scss';
+
+/**
+ * Component to display a budget badge with the given amount.
```

Comment:
@ankitrox We are not really displaying the amount here but rather the percentage increase/decrease.
```suggestion
 * Renders a badge displaying the percentage change in budget.
```